site/content/docs/4.3/components/toasts.md

```diff
@@ -98,6 +98,60 @@ When you have multiple toasts, we default to vertically stacking them in a reada
 </div>
 {{< /example >}}
 
+### Custom content
+
+Customize your toasts by removing sub-components, tweaking with [utilities]({{< docsref "/utilities/api" >}}), or adding your own markup. Here we've created a simpler toast by removing the default `.toast-header`, adding a custom hide icon from [Bootstrap Icons](https://icons.getbootstrap.com), and using some [flexbox utilities]({{< docsref "/utilities/flex" >}}) to adjust the layout.
+
+{{< example class="bg-light" >}}
+<div class="toast d-flex" role="alert" aria-live="assertive" aria-atomic="true">
+  <div class="toast-body">
+    Hello, world! This is a toast message.
+  </div>
+  <button type="button" class="ml-auto p-2 close" data-dismiss="toast" aria-label="Close">
+    <span aria-hidden="true">
+      <svg class="bi bi-x" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We should use what we currently use everywhere, `&times;`

Reply:
The point of the example here is to show a different icon :).

Reply:
I'm 50-50 on this. I mean, it's generally pretty straight-forward, I'm not sure it needs an example...

Reply:
Also, I'd say the extra `span` is not needed; you should move `aria-hidden` to the SVG itself.